src/module-utils.js

```diff
@@ -0,0 +1,21 @@
+// a require function with both ES5 and ES6 default export support
+function requireX(path) {
```

Comment:
I think a more descriptive name should be used. When someone sees this function called in the code they should have some idea of what it does and why it is used.

Something like: `requirePossibleEsModule` or `requireDefault`

Reply:
I wanted the name to be concise. I could not think of any descriptive name 😄. I am open to all suggestions.

Reply:
I like `requireDefault`. It is not too long and fairly descriptive of what it does.

Reply:
What about `requireModule`? My problem with `requireDefault` is that this may not use "default", and so the name becomes a little misleading.

Reply:
sounds good to me

Reply:
Fixed in https://github.com/atom/atom/pull/21112/commits/2ea54ac3b284cb4b9b7c667de350c5b79fb51b9e